doc/thingpedia-tutorial-hello-world.md

```diff
@@ -0,0 +1,80 @@
+# Tutorial 0: Hello World
```

Comment:
Should we renumber the tutorials? Or are we hard-core programmers?

Reply:
Haha sure, I have no preference. I can renumber it